Scripts/Python/clftYeeshaPage08.py

```diff
@@ -147,32 +150,26 @@ def OnGUINotify(self,id,control,event):
         if isinstance(control,ptGUIControlButton):
             btnID = control.getTagID()
 
-        if event == kShowHide:
-            if control.isEnabled():
-                #control.show()
-                if self.GotPage():
-                    mydialog = PtGetDialogFromString(DialogName)
-                    ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage08)).disable()
-
-        elif event == kAction and btnID == kYeeshaPage08:
+        if event == kAction and btnID == kYeeshaPage08:
             PtDebugPrint("DEBUG: clftYeeshaPage08.OnGUINotify():\tPicked up page")
 
             RespClose.run(self.key)
             isOpen = 0
             PtHideDialog(DialogName)
             self.SetStdGUIVisibility(1)
+            RespGlow.run(self.key, state='GlowSound')
     
             if self.GotPage(): 
                 PtDebugPrint ("DEBUG: clftYeeshaPage08.py: You've already found Yeesha Page #8. Move along. Move along.")
                 return
             else:
                 PtDebugPrint ("DEBUG: clftYeeshaPage08.py: Yeesha Page #8 is new to you.")       
-                PtDebugPrint ("DEBUG: clftYeeshaPage08.py: Trying to update the value of the SDL variable %s to 1" % ("YeeshaPage8"))
+                PtDebugPrint ("DEBUG: clftYeeshaPage08.py: Trying to update the value of the SDL variable %s to 4" % ("YeeshaPage8"))
                 vault = ptVault()  
                 psnlSDL = vault.getPsnlAgeSDL()
                 if psnlSDL:
                     YeeshaPageVar = psnlSDL.findVar("YeeshaPage8")    
-                    YeeshaPageVar.setInt(1)
+                    YeeshaPageVar.setInt(4)
```

Comment:
It's very unclear to me what this code is doing. It does not appear to be related to the page additions, as page 8 is a very old page. Could you please 1) explain what this is for and 2) split these out to a separate PR so they can be tracked as a fix for whatever this is fixing and merged independently of these new pages?

Reply:
I believe it has to do with this? #1147 
Not sure why it's repeated in this PR though since the other was merged.

Reply:
Ah, that makes sense, thanks for tracking that down, Doobes! @Hazado then I think it would make sense for you to rebase all the new Relto page PRs to make sure they don't include all these unrelated changes, right?

Reply:
Heh, that's something I try to make a habit, but I'll admit, I forget sometimes too, particularly for existing branches. 😛